tests/test_checker.py

```diff
@@ -52,6 +54,66 @@
 NUM_NEW_VERSIONS = 2
 
 
+async def _test_update(
```

Comment:
This is clearly a method, not a standalone function. No point to separate it from the test class. To make it reusable, attach it to a common ancestor of `TestExternalDataChecker` and `TestImportantGitExternalDataChecker`, somehow like this:
```python
class _TestWithInlineManifest(unittest.IsolatedAsyncioTestCase):
    async def _test_update(self, ...):
        ...

class TestExternalDataChecker(_TestWithInlineManifest):
    ...

class TestImportantGitExternalDataChecker(_TestWithInlineManifest):
    ...
```
Then you can call `self._test_update()` on both.

Reply:
thanks, fixed